engine/src/main/java/org/terasology/rendering/gltf/GLTFMeshFormat.java

```diff
@@ -86,9 +91,71 @@ public MeshData load(ResourceUrn urn, List<AssetDataFile> inputs) throws IOExcep
             checkIndicesBuffer(indicesBuffer);
 
             readBuffer(loadedBuffers.get(indicesBuffer.getBuffer()), indicesAccessor, indicesBuffer, mesh.getIndices());
+            applyTransformations(gltf, mesh.getVertices(), mesh.getNormals());
             return mesh;
         }
     }
 
+    private void applyTransformations(GLTF gltf, TFloatList vertices, TFloatList normals) {
+        GLTFNode node = null;
+        for(GLTFNode curr : gltf.getNodes()) {
```

Comment:
Minor whitespace style issues here and elsewhere, make sure you have Checkstyle hooked up and run it in the IDE against modified files 👍

Reply:
I fixed the issues in the file I modified, should I fix the rest of the GLTF checkstyle issues as well or leave those for now?

Reply:
Thanks! Generally if you touch a file feel free to fix other uncontroversial style fixes within it, so long as that wouldn't overwhelm the diff (make it hard to spot your true changes). Same with related files, but the further they get from the core changes and the more changes overall the more likely it'll be a better idea to do it in a different PR. Bit of a judgement call case by case.

In this case you're adding to an unmerged PR so it may be wise to avoid more change in case the other PR gets modified again. But any _new_ code you introduce at least wouldn't add any more diffs to style right :-)

tl;dr: I'd leave the rest as-is in this case :)